src/routing/uploadGCPRouter.js

```diff
@@ -0,0 +1,52 @@
+import Router from 'koa-router';
+import uuidV4 from 'uuid/v4';
+import Storage from '@google-cloud/storage';
+
+export default class GoogleUploadRouter extends Router {
+  //pass in googleConfig, trace this back 
+  constructor(googleConfig) {
+    console.log(googleConfig)
+    super();
+    this.bucket = googleConfig.GCP_Bucket;
+    this.storage = new Storage({
+      credentials: {
+        "client_email": googleConfig.client_email,
```

Comment:
I feel like there should be an access_id or something, not just email but I also could be very wrong here so